api/src/main/java/org/apache/apex/api/users/ControlTuple.java

```diff
@@ -16,15 +16,15 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package org.apache.apex.api;
+package org.apache.apex.api.users;
```

Comment:
Would engine be doing any special handling of these batch control tuples compared to other control tuples? If not can these be in malhar?